asclepius/README.md

```diff
@@ -115,7 +118,8 @@ HTTP Endpoints
     "label": "torasemide/trandolapril DDI",
     "uri": "http://purl.obolibrary.org/obo/DINTO_11031",
     "harmful": false,
-    "spacing": 3
+    "spacing": 2,
```

Comment:
No, that's never displayed in the UI, it's equivalent to good/bad in terms of the mock data